packages/legacy/core/App/screens/PINEnter.tsx

```diff
@@ -366,15 +378,29 @@ const PINEnter: React.FC<PINEnterProps> = ({ setAuthenticated, usage = PINEntryU
       )
     }
 
-    return <Text style={style.helpText}>{t('PINEnter.EnterPIN')}</Text>
-  }, [store.lockout.displayNotification, style.helpText, t, biometricsEnrollmentChange, biometricsErr])
+    return (
+      <>
+        <Text style={style.title}>{t('PINEnter.Title')}</Text>
+        <Text style={style.subTitle}>{t('PINEnter.SubText')}</Text>
+      </>
+    )
+  }, [
+    store.lockout.displayNotification,
+    style.helpText,
+    t,
+    biometricsEnrollmentChange,
+    biometricsErr,
+    style.subTitle,
+    style.title,
+  ])
 
   return (
     <KeyboardView>
       <View style={style.screenContainer}>
         <View style={style.contentContainer}>
-          <Image source={Assets.img.logoSecondary.src} style={style.image} />
+          {/* <Image source={Assets.img.logoSecondary.src} style={style.image} /> */}
```

Comment:
Why was the image commented out? Should it be removed entirely or uncommented? @FarjadGov14